pkg/codegen/go/gen.go

```diff
@@ -5067,7 +5048,10 @@ func GeneratePackage(tool string,
 	}
 
 	// create a go.mod file with references to local dependencies
-	if pkg.SupportPack {
+	//
+	// Skip if Parameterization is enabled, as we don't want to generate a go.mod
+	// file in that case (implies this is a subpackage already).
```

Comment:
This comment reads wrong. We're only outputting a go.mod if either supportPacck is true OR we're parameterised ("If the package is parameterized generate a go.mod for it" the old comment from before).

Reply:
not only did the comment read wrong but I actually had the wrong boolean operator I think